crossai/pipelines/audio.py

```diff
@@ -16,3 +16,4 @@ def __init__(self, X):
 
         self.data = X.data
         self.labels = X.label
+        self.filename = X.filename
```

Comment:
Additionally, ensure that filenames are consistently propagated when performing the sliding window technique.